isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -1194,6 +1194,339 @@ End)");
+
+  std::istringstream scconfigStr(R"(Object = SensorParameters
+    Group = APOLLO15/METRIC
+      CKDegree=2
+      CKSolveDegree=2
+      CamSolve=Angles
+      Twist=yes
+      SPSolve=positions
+      Spacecraft_position_sigma=1000.0
+      Camera_Angles_Sigma=2.0
+    EndGroup
+    Group = "LUNARRECONNAISSANCEORBITER/NACR"
+      CamSolve=accelerations
+      Twist=yes
+      OverExisting=yes
+      SPSolve=accelerations
+      overhermite=yes
+      Spacecraft_position_sigma=100.0
+      Spacecraft_velocity_sigma=1.0
+      Spacecraft_acceleration_sigma=0.1
+      Camera_Angles_Sigma=2.0
+      Camera_AngularVelocity_Sigma=0.1
+      Camera_AngularAcceleration_Sigma=0.01
+    EndGroup
+    Group = "LUNARORBITER5/HIGHRESOLUTIONCAMERA"
+      CamSolve=angles
+      Twist=yes
+      Camera_Angles_Sigma=2.0
+      SPSolve=positions
+      Spacecraft_position_sigma=1000.0
+    EndGroup
+  EndObject
+  End)");
+
+  Pvl scconfig;
+  scconfigStr >> scconfig;
+  scconfig.write(scconfigPath);
+
+  QVector<QString> args = {"fromlist="+cubeListFile, "cnet="+controlNetPath,
+                           "onet="+outCnetFileName, "heldlist="+heldlistpath, "scconfig="+scconfigPath,
+                           "radius=true", "point_radius_sigma=50",
+                           "bundleout_txt=no", "output_csv=no",
+                           "residuals_csv=no", "file_prefix="+prefix.path()+"/"};
+
+ UserInterface options(APP_XML, args);
+
+ try {
+   jigsaw(options);
+ }
+ catch (IException &e) {
+   FAIL() << "Failed to bundle: " << e.what() << std::endl;
+ }
+
+ CSVReader heldHeader = CSVReader(prefix.path() + "/bundleout_images_held.csv",
+                              false, 0, ',', false, true);
+
+ CSVReader::CSVAxis csvLine = heldHeader.getRow(2);
+ // sample res
+ EXPECT_NEAR(csvLine[1].toDouble(), 7.75E-10, 0.0001);
+ // line res
+ EXPECT_NEAR(csvLine[2].toDouble(), 6.87E-10, 0.0001);
+ // total res
+ EXPECT_NEAR(csvLine[3].toDouble(), 7.32E-10, 0.0001);
+ // final X
+ EXPECT_NEAR(csvLine[6].toDouble(), 657.4955708, 0.0001);
+ // final Y
+ EXPECT_NEAR(csvLine[11].toDouble(), 1126.218168, 0.0001);
+ // final Z
+ EXPECT_NEAR(csvLine[16].toDouble(), 1301.314774, 0.0001);
+ // final RA
+ EXPECT_NEAR(csvLine[21].toDouble(), 118.2110421, 0.0001);
+ // final DEC
+ EXPECT_NEAR(csvLine[26].toDouble(), 53.3173841, 0.0001);
+ // final TWIST
+ EXPECT_NEAR(csvLine[31].toDouble(), -150.2844053, 0.0001);
+
+ // assert corrections are very small
+ // X Correction
+ EXPECT_LE(std::abs(csvLine[5].toDouble()), 1e-10);
+ // Y Correction
+ EXPECT_LE(std::abs(csvLine[10].toDouble()), 1e-10);
+ // Z Correction
+ EXPECT_LE(std::abs(csvLine[15].toDouble()), 1e-10);
+ // RA Correction
+ EXPECT_LE(std::abs(csvLine[20].toDouble()), 1e-10);
+ // DEC Correction
+ EXPECT_LE(std::abs(csvLine[25].toDouble()), 1e-10);
+ // TWIST Correction
+ EXPECT_LE(std::abs(csvLine[30].toDouble()), 1e-10);
+
+
+ CSVReader header = CSVReader(prefix.path() + "/bundleout_images_APOLLO15_METRIC.csv",
```

Comment:
The scconfigHeld test only outputted bundleout_images_APOLLO15_METRIC.csv and the hold.csv. It did not include the other instruments from my scconfig file. Again, not sure if this is due to using the Apollo network fixture or needing to make changes in my scconfig file.